hs_app_netCDF/models.py

```diff
@@ -156,6 +156,19 @@ def can_have_multiple_files(cls):
         # can have only 1 file
         return False
 
+    # add resource-specific HS terms
+    def get_hs_term_dict(self):
+        # get existing hs_term_dict from base class
+        hs_term_dict = super(NetcdfResource, self).get_hs_term_dict()
+        # add new terms for NetCDF res
+        hs_term_dict["HS_NETCDF_FILE_NAME"] = ""
+        for res_file in self.files.all():
+            fn = res_file.resource_file.name
```

Comment:
I think for 'federated' files this won't work. Use the ```get_resource_file_extension(res_file)``` to get the file extension. Similar logic would apply to get the name of the file. @hyi may correct me on this.